src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                </tr>
+                <tr>
+                    <td>1-2</td>
+                    <td>Status Data ID</td>
+                </tr>
+                <tr>
+                    <td>3-7</td>
+                    <td>Data (signed 32bit)</td>
+                </tr>
+                <tr>
+                    <td>8</td>
+                    <td>Data Length</td>
+                </tr>
+                <tr>
+                    <td>9-10</td>
+                    <td>Allowed Error</td>
+                </tr>
+                <tr>
+                    <td>11-12</td>
+                    <td>Estimate Error</td>
+                </tr>
+                <tr>
+                    <td>12-13</td>
+                    <td>Deprecated = 0</td>
+                </tr>
+                <tr>
+                    <td>14</td>
+                    <td>Smoothing Coefficient</td>
+                </tr>
+            </tbody>
+        </table>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Curve Function</td>
+                    <td>
+                        2 = Add Point
+                        <br></br>
+                        3 = Save Curve
+                    </td>
+                </tr>
+                <tr>
+                    <td>Allowed Error</td>
+                    <td>Vertical distance threshold (must be &gt; 0). All points with their vertical distance &gt; threshold are declared as significant points and saved.</td>
+                </tr>
+                <tr>
+                    <td>Estimate Error</td>
+                    <td>If EstimateError &gt; 0 and the new point deviated from estimated value &gt; EstimateError, we will reduce/save the curve.</td>
+                </tr>
+                <tr>
+                    <td>Smoothing Coefficient</td>
+                    <td>Applies a low pass filter to the data.
+                        <br></br>
+                        0 = No filtering
+                        <br></br>
+                        1-254 = Smoothing coefficient magnitude
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Type 12 Logging With Timestamp</h4>
+        <p>This message can be used to send status data with a timestamp. Possible use cases:</p>
+        <ol>
+            <li>Store data in the IOX while the GO device is asleep and send all data after waking up</li>
+            <li>
+                Run the curve logging algorithm in the IOX and send those points to be transmitted to MyGeotab. 
+                <br></br>
+                Additional information about curve logging can be found here: <a href="https://github.com/Geotab/curve" target="_blank" rel="noreferrer">Curve Logging</a>
+            </li>
+        </ol>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on